scripts/init.sh

```diff
@@ -154,37 +147,28 @@ case $1 in
             exit 1
         fi
 
-        log_success_msg "Starting the process" "$name"
+        log_success_msg "Starting the process" "$NAME"
         if which start-stop-daemon > /dev/null 2>&1; then
-            start-stop-daemon --chuid $GROUP:$USER --start --quiet --pidfile $pidfile --exec $daemon -- -pidfile $pidfile -config $config $INFLUXD_OPTS >>$STDOUT 2>>$STDERR &
+            start-stop-daemon --chuid $GROUP:$USER --start --quiet --pidfile $PIDFILE --exec $DAEMON -- -pidfile $PIDFILE -config $CONFIG $INFLUXD_OPTS >>$STDOUT 2>>$STDERR &
         else
-            su $USER -c "nohup $daemon -pidfile $pidfile -config $config $INFLUXD_OPTS >>$STDOUT 2>>$STDERR &"
-        fi
-
-        wait_for_startup && is_process_running
-        if [ $? -ne 0 ]; then
-            log_failure_msg "$name process failed to start"
-            exit 1
-        else
-            log_success_msg "$name process was started"
-            exit 0
+            nohup $DAEMON -pidfile $PIDFILE -config $CONFIG $INFLUXD_OPTS >>$STDOUT 2>>$STDERR &
```

Comment:
Is the removal of `su` here an intentional change? Now, on CentOS 6 and 7, where start-stop.daemon is not available, InfluxDB runs as root if started via init.d script. Prepending `sudo -g $GROUP -u $USER` to this line is a workaround.